tests/ref/test_schema.py

```diff
@@ -48,7 +39,7 @@ class TestIsolatePlan:
     def test_create_plan_from_records(
```

Comment:
Split this class-based test into two method: `test_monopartite` and `test_multipartite`.